configs/objectiv.json

```diff
@@ -20,7 +20,7 @@
     "lvl3": "article h3",
     "lvl4": "article h4",
     "lvl5": "article h5, article td:first-child",
-    "text": "article p, article li, article td:last-child"
+    "text": "article p, article li:not(.ignore-for-search), article td:last-child"
```

Comment:
We have the `selectors_exclude` option so you can better read what you index/what you exclude: https://docsearch.algolia.com/docs/legacy/config-file/#selectors_exclude-optional

Reply:
That looks like a great solution for us, thank you! Will update this PR and let you know.

Reply:
PR updated. Would be great if you could merge and trigger a crawl. Thanks!